src/components/chat/ModalContainer/ModalContent/ConfirmationDeleting.tsx

```diff
@@ -1,10 +1,34 @@
+import { useRouter } from "next/navigation";
 import { ModalContentType } from "../ModalContainer";
+import { deleteChat } from "@/app/[lang]/(protected)/chat/delete/deleteChat";
 
 export default function ConfirmationDeleting({
   openModal,
+  chatRoomUuid,
+  lang
 }: {
   openModal: (content: ModalContentType) => void;
+  chatRoomUuid: string;
+  lang: string
 }) {
+  const router = useRouter();
+
+  const handleDelete = async () => {
+    if (chatRoomUuid) {
+      const deleted = await deleteChat(chatRoomUuid);
+      if (deleted) {
+        openModal("CompleteDeleting");
+        setTimeout(() => {
+          openModal(null);
+          router.push(`/${lang}/chat/all`);
+          router.refresh()
+        }, 2000);
+      } else {
+        openModal(null);
+      }
+    }
+  };
```

Comment:
You can remove the if condition to check if a `chatRoomUuid` is a true value, because as we know the chat id should be valid